### PR TITLE
MOMUG-125: Updating jsbox translations

### PR DESCRIPTION
**Pull request by bruskiza**

### CODE REVIEW

**Comment by bruskiza** [comment]
@rudigiesler @KaitCrawford @smn review please.

**Comment by bruskiza** [comment]
Right @rudigiesler here you go.
Review?

**Comment by bruskiza** [comment]
@rudigiesler @KaitCrawford ready for review.

**Comment by bruskiza** [comment]
@rudigiesler @KaitCrawford review please.

File: config/go-app-ussd_public.cgg_UG.json [hunk 1]
```diff
@@ -0,0 +1,402 @@
...
+    ],
+    "Thank you. The number which receives messages has been updated.": [
+        null,
+        "Yebaare, enamba yokwakiira obutumwa yayerezibwa."
+    ],
+    "Why do you no longer want to receive messages?": [
+        null,
+        "Ahabwenki otakyayeenda kutunga obutumwa?"
+    ],
+    "We are sorry for your loss. Would you like to receive a small set of free messages from FamilyConnect that could help you in this difficult time?": [
```
Comment (rudigiesler) [inline config/go-app-ussd_public.cgg_UG.json]: This hasn't yet been updated from the copy update here: https://github.com/praekelt/chms-jsbox/pull/56/files#diff-f11efc6ebde289613436f88de0401d97L1086

File: config/go-app-ussd_public.cgg_UG.json [hunk 2]
```diff
@@ -0,0 +1,402 @@
...
+    ],
+    "Family member": [
+        null,
+        "Omuntu ondijo owomuka omu"
+    ],
+    "Trusted friend": [
+        null,
+        "Munywani wawe omwesigwa"
+    ],
+    "Mother miscarried": [
```
Comment (rudigiesler) [inline config/go-app-ussd_public.cgg_UG.json]: This hasn't been updated along with the copy here: https://github.com/praekelt/chms-jsbox/pull/56/files#diff-f11efc6ebde289613436f88de0401d97L1536

**Comment by smn** [comment]
:+1:

**Pull request merged**